velox/exec/tests/RowContainerTest.cpp

```diff
@@ -827,78 +827,85 @@ TEST_F(RowContainerTest, probedFlag) {
   // No 'probed' flags set. Verify all false, except for null key row.
   auto result = BaseVector::create<FlatVector<bool>>(BOOLEAN(), 1, pool());
   rowContainer->extractProbedFlags(
-      rows.data(), size, false /*replaceFalseWithNull*/, result);
+      rows.data(),
+      size,
+      true /*nullForNullKeys*/,
```

Comment:
Consider this format as suggested in Google coding style for such function input annotations:
```
/*nullForNullKeys=*/true,
```

Reply:
This would deviate from the current style. If we want to switch to this, let's open a PR to update coding style first.